modules/backend/widgets/lists/partials/_list_pagination_simple.htm

```diff
@@ -29,7 +29,8 @@
         <select
             disabled
             name="page"
-            class="form-control input-sm custom-select select-no-search">
+            class="form-control input-sm custom-select select-no-search"
+            autocomplete="off">
```

Comment:
Why does this need to be added?

Reply:
Its to fix long standing issue when browser caches input values after you refresh page.

Reply:
Same issue is with checkboxes in list but thats for separate PR